<launch>
  <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="AggregatePoints" args="load lidar_pkg/AggregatePoints standalone_nodelet" output="screen">
    <remap from="/AggregatePoints/agg_points" to="MrfGroundSeg/agg_points"/> -->
  </node>

  <node pkg="nodelet" type="nodelet" name="MrfGroundSeg" args="load lidar_pkg/MrfGroundSeg standalone_nodelet" output="screen">
    <remap from="/MrfGroundSeg/mrf_filtered_points" to="OccupancyGridGeneration/mrf_filtered_points"/>
  </node>

  <node pkg="nodelet" type="nodelet" name="OccupancyGridGeneration" args="load lidar_pkg/OccupancyGridGeneration standalone_nodelet" output="screen">
    <remap from="/OccupancyGridGeneration/occupancy" to="InferenceTF/occupancy"/>
    <remap from="/OccupancyGridGeneration/masses" to="InferenceTF/masses"/>
  </node>

 <node pkg="nodelet" type="nodelet" name="InferenceTF" args="load lidar_pkg/InferenceTF standalone_nodelet" output="screen">
    <remap from="/InferenceTF/prediction_all" to="Visualize/prediction_all"/>
  </node>

  <node pkg="nodelet" type="nodelet" name="Visualize" args="load lidar_pkg/Visualize standalone_nodelet" output="screen">
  </node>

</launch>
